application/views/scripts/index/index.phtml

```diff
@@ -13,5 +13,5 @@
       <td><a target="graph_iframe" class="icon-clock" href="<?= urldecode($this->url) ?>&from=-1y">1 year</a></td>
     </tr>
   </table>
-  <iframe name ="graph_iframe" src="<?= urldecode($this->url) ?>&from=-1d" style="height: 700px; width: 800px" frameborder="no" scrolling="no" marginwidth="0" marginheight="0" hspace="0" vspace="0"></iframe>
+  <iframe name ="graph_iframe" src="<?= urldecode($this->url) ?>&from=-1d" style="height: 800px; width: 1024px" frameborder="no" scrolling="no" marginwidth="0" marginheight="0" hspace="0" vspace="0"></iframe>
```

Comment:
Good idea.  I have this added and just need to do a little more testing before I commit it.  This should be ready to go later tonight or tomorrow.